beacon-chain/db/kv/kv.go

```diff
@@ -150,6 +181,7 @@ func NewKVStore(ctx context.Context, dirPath string, config *Config) (*Store, er
 			stateSlotIndicesBucket,
 			blockParentRootIndicesBucket,
 			finalizedBlockRootsIndexBucket,
+			BlockRootValidatorKeysIndexBucket,
```

Comment:
This bucket stores indices for the` validator entry` bucket. Thus the name.

Reply:
I don't want to be *too* picky but... :wink:

There is a concept of validator index, which is the position of a validator in the validator list. There is also a concept of a validator public/private key. You store hashes, which are not the same as validator indexes nor keys. That's why I would call this bucket `BlockRootValidatorHashes`, so that other devs do not get confused. Would you agree it makes sense?

Reply:
I get your point. I rename the bucket to `BlockRootValidatorHashes. Also i will change the name  in the document from`Validator Index` to `BlockRootValidatorHashes`